CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing to Tenant Security Client Go
+
+## Tests
+
+This client has both a set of unit tests as well an integration test suite. Because of the complexity of the various services required to run non-unit test suites, these tests require additional setup, which is explained below.
+
+### Unit Tests
+
+Tests that check functionality that is contained within the client.
+
+```
+go test
```

Comment:
Arguably this should be `go test ./...` to test all the packages in this repo instead of just the root one. But since we only have tests at the root, it's equivalent.

Reply:
Sure I can change that